easybuild/easyconfigs/b/Boost/Boost-1.65.1-intel-2017a-Python-3.6.3.eb

```diff
@@ -0,0 +1,24 @@
+name = 'Boost'
+version = '1.65.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.boost.org/'
+description = "Boost provides free peer-reviewed portable C++ source libraries."
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+toolchainopts = {'usempi': True, 'pic': True}
+
+source_urls = ['https://sourceforge.net/projects/%(namelower)s/files/%(namelower)s/%(version)s']
+sources = ['%%(namelower)s_%s.tar.gz' % '_'.join(version.split('.'))]
+checksums = ['a13de2c8fbad635e6ba9c8f8714a0e6b4264b60a29b964b940a22554705b6b60']
+
+dependencies = [
+    ('bzip2', '1.0.6'),
+    ('zlib', '1.2.11'),
+    ('Python', '3.6.3'),
```

Comment:
@damianam I think this dep is missing?

Reply:
Just bumping to `intel/2017b` would fix this...

Reply:
Argh, damnit , I thought I was using `intel/2017b`. Bumping it.........